cl/users/templates/profile/nav.html

```diff
@@ -54,6 +54,10 @@
                 role="presentation">
                 <a href="{% url "view_api" %}"><i class="fa fa-code gray"></i>&nbsp;API</a>
             </li>
+            <li class="{% block nav-webhooks%}inactive{% endblock %}"
+                role="presentation">
+                <a href="{% url "view_webhooks" %}"><i class="fa fa-code gray"></i>&nbsp;Webhooks</a>
+            </li>
```

Comment:
We should wrap this in a django-waffle [flag](https://waffle.readthedocs.io/en/stable/types/flag.html) and then we can turn it off and on for various users until it's ready to launch. Something like this should do the trick. 

At the top of this file:
```
{% load waffle_tags %}
```

Then wrap your thing in it:

```suggestion
            {% flag "user-webhooks" %}
            <li class="{% block nav-webhooks%}inactive{% endblock %}"
                role="presentation">
                <a href="{% url "view_webhooks" %}"><i class="fa fa-code gray"></i>&nbsp;Webhooks</a>
            </li>
            {% endflag %}
```

When we deploy this won't be visible by default and a flag will be created in the DB. Once we're ready to launch, we can set the flag to false to let people see it and we can delete this code (someday). You can play with flags if you want via the admin site.

Reply:
Oh, and this won't break the link; it just hides the tab, so if you are working and want to see the page, you of course can just URL hack your way to it.

Reply:
@mlissner Smart man. You clearly know how to run an open source project with lots of contributors.

Reply:
Well, I just landed the waffle feature last week, so I guess I'm getting there. :)